### DISCUSSION TITLE
Batch Create Extensions or import CSV.

### DISCUSSION THREAD

**Discussion by sumonchai** [mikopbx/Core#670] [Ideas (Идеи)]
Next time Have Future ?
![image](https://github.com/mikopbx/Core/assets/5920739/d84fcc2f-206a-485a-868f-297c25ce7731)

**Comment by boffart** [comment]
https://wiki.mikopbx.ru/module-backup#zagruzka_dannyx_iz_csv
Use Google Translator to translate the article
